doc/onboarding.md

```diff
@@ -136,26 +145,30 @@ onboarding session.
 
 ## Exercise: Make a PR adding yourself to the README
 
-  * Example: [https://github.com/nodejs/node/commit/7b09aade8468e1c930f36b9c81e6ac2ed5bc8732](https://github.com/nodejs/node/commit/7b09aade8468e1c930f36b9c81e6ac2ed5bc8732)
-    * For raw commit message: `git log 7b09aade8468e1c930f36b9c81e6ac2ed5bc8732 -1`
+  * Example: [https://github.com/nodejs/node/commit/ce986de829457c39257cd205067602e765768fb0](https://github.com/nodejs/node/commit/ce986de829457c39257cd205067602e765768fb0)
+    * For raw commit message: `git log ce986de829457c39257cd205067602e765768fb0 -1`
   * Collaborators are in alphabetical order by GitHub username.
   * Optionally, include your personal pronouns.
   * Label your pull request with the `doc` subsystem label.
   * Run CI on your PR.
-  * After a `LGTM` or two, land the PR.
+  * After one or two approvals, land the PR.
     * Be sure to add the `PR-URL: <full-pr-url>` and appropriate `Reviewed-By:` metadata!
+    * [`core-validate-commit`][] helps a lot with this – install and use it if you can!
+    * If you use Chrome, [`node-review`][] fetches the metadata for you
 
 ## Final notes
 
-  * don't worry about making mistakes: everybody makes them, there's a lot to internalize and that takes time (and we recognize that!)
-  * very few (no?) mistakes are unrecoverable
-  * the existing collaborators trust you and are grateful for your help!
-  * other repos:
-    * [https://github.com/nodejs/dev-policy](https://github.com/nodejs/dev-policy)
-    * [https://github.com/nodejs/NG](https://github.com/nodejs/NG)
-    * [https://github.com/nodejs/api](https://github.com/nodejs/api)
+  * Don't worry about making mistakes: everybody makes them, there's a lot to internalize and that takes time (and we recognize that!)
+  * No mistakes are unrecoverable!
```

Comment:
I have no suggestions on how to say this succinctly... but it may be worth adding a reminder that deleting the fork for a PR removes the commit history from that PR (that is, the PR will show 0 commits / 0 changes)